lib/utils/net-utils.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const Promise = require('bluebird')
+const net = require('net')
+
+const NetUtils = {
+  isPortAvailable (port, listenAddress) {
+    return new Promise((resolve, reject) => {
+      const server = net.createServer()
+
+      server.unref()
+      server.on('error', (err) => {
+        server.close()
```

Comment:
I suppose it's redundant

Reply:
Are you sure ? I see in nodejs documentation (a bit above https://nodejs.org/api/net.html#net_server_listen_handle_backlog_callback) that author invoke `server.close()` inside `server.on('error', (e) => {})` - to be honest I don't know why and whether it's necessary.

Reply:
https://nodejs.org/api/net.html#net_event_error
well, it seems reasonable to leave it